theatre/studio/src/selectors.ts

```diff
@@ -82,3 +87,16 @@ export function getSelectedSequence(): undefined | Sequence {
 
   return sheet.getSequence()
 }
+
+export function getCopiedKeyframes(): ISelectedKeyframes {
+  return val(getStudio()!.atomP.ahistoric.keyframesClipboard) || []
+}
+
+export function getTracks(
```

Comment:
A better alternative already exists, at either [`getMapOfValidSequenceTracks_forStudio()`](https://github.com/theatre-js/theatre/blob/6fac2d71e977f41eb3c1a0e0e8fb7e73a3d9d89b/theatre/core/src/sheetObjects/SheetObjectTemplate.ts#L161) or [`getArrayOfValidSequenceTracks()`](https://github.com/theatre-js/theatre/blob/6fac2d71e977f41eb3c1a0e0e8fb7e73a3d9d89b/theatre/core/src/sheetObjects/SheetObjectTemplate.ts#L111)

Reply:
I used `getArrayOfValidSequenceTracks` [here](https://github.com/theatre-js/theatre/pull/65/files#diff-dbaec4167502b68fe66ce629dc407b3ad704643ab20c769560ee2c569cd9a7c5R244), but still had to access this data the same way [here](https://github.com/theatre-js/theatre/pull/65/files#diff-e4182ccc7879dade9478e9aff683c5b7499389bc7547e797f029752ae88db5aaR63) as I needed the `trackData`.